Dockerfile

```diff
@@ -1,17 +1,5 @@
-FROM registry.gitlab.com/famedly/infra/containers/rust:main as builder
+FROM rust:1 as builder
```

Comment:
Not using our own image here because it's public repo, or?

Reply:
yes, OSS should be possible to build without us right? Somebody complained in GitLab back in the day :D